nntrainer/tensor/memory_pool.cpp

```diff
@@ -122,18 +133,22 @@ void MemoryPool::allocate() {
  *
  */
 std::shared_ptr<MemoryData> MemoryPool::getMemory(unsigned int idx) {
-  if (mem_pool == nullptr)
-    throw std::invalid_argument("Getting memory before allocation");
+  // if (mem_pool == nullptr)
+  //   throw std::invalid_argument("Getting memory before allocation");
 
-  char *ptr = static_cast<char *>(mem_pool) + memory_offset.at(idx - 1);
+  // char *ptr = static_cast<char *>(mem_pool) + memory_offset.at(idx - 1);
   // void *ptr;
   // allocators.at("qnn")->alloc(ptr, memory_size.at(idx-1), 1);
-  // std::cout << idx << " : "<<std::endl;
-  // if(ptr == nullptr)
-  //   std::cout << "alloc failed"<<std::endl;
 
-  auto mem_data = std::make_shared<MemoryData>((void *)ptr);
+#ifdef ENABLE_QNN
+  auto mem_data = std::make_shared<MemoryData>((void *)memory_ptrs.at(idx - 1));
+#else
```

Comment:
`#if defined(ENABLE_QNN)` also `calloc`'ed `mem_pool`, but here result of that allocation is not checked

Reply:
That's right Thank you for review